File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let nums = s.split(" ").map(Number); // 문자열을 Number 타입으로 바꿔서 변수에 저장
    return Math.min(...nums) + " " + Math.max(...nums); // 최소값 + " " + 최대값 반환
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] str = s.split(" ");
        ArrayList<Integer> tmp = new ArrayList<>();
        
        for (String i : str){
            tmp.add(Integer.parseInt(i));
        }
        
        int min = Collections.min(tmp);
        int max = Collections.max(tmp);
        
        answer = Integer.toString(min) + " " + Integer.toString(max);
        return answer;
    }
}